### DISCUSSION TITLE
From the NFL to Software Engineer at GitHub

### DISCUSSION THREAD

**Discussion by sgoldfarb2** [community/community#63383] [Programming Help]
I loved my job as a video editor. 

I met incredibly cool celebrities, worked on really interesting projects, and had the opportunity to work at incredible companies. I started my career at the NFL, went to NBCUniversal, and lived the startup life after, freelance editing video for years for TheMuse, helping prospective employees find out more about employers they were interested in working for.

Yet, I wasn’t fulfilled. 

There were a few reasons. The first was knowing you were replaceable at any moment. Who wouldn’t want to work for the NFL? On top of that, it just wasn’t sustainable. I found myself needing two or three jobs at a time to pay all my bills.

I needed a change.

It took five years of self-learning mixed with night classes and a bootcamp to make the switch, but finally, I became a software engineer.

For me, the right bootcamp was Codesmith. For no bigger reason other than I loved their teaching style and the community they had built. Not only was I able to hone my technical skills at a higher level, but I made friendships that have changed my life. For anyone interested in becoming a software engineer, a bootcamp is absolutely not necessary. But if you, like me, find it hard to stay focused and figure out the right next step to learn, they can be an incredibly helpful solution.

I often get asked about making the switch after 20/25/30/etc. I started to learn to code when I was 25, and I formally made the career change around the age of 30.

I try to be honest. As much as I shouldn’t have been, I used to be embarrassed to ask people a decade younger than myself for help at work. The career change taught me a lot of humility. 

With that being said, the best part of making the career change was feeling like I was finally doing something I loved and that it had a sustainable future to it. There is so much room for growth and now, I work a second job teaching because I want to, not because I need to pay my bills.

I’ll never forget my past. Being a video editor means I understand clients, products, visual design, and keeping brand consistency really well. I understand how to take nothing and turn it into something beautiful that maintains a brand’s voice. Because of that experience, I am able to think from a completely different perspective than some of my teammates, and that in itself is an amazing thing.

Now that I’ve been in this career for a few years, I’m focused on giving back to the community and helping others who want to make a career change. I teach classes on weekends. I have a discord server where people can ask questions - technical or otherwise. Every time a former student sends me a message about accepting their first role in the industry, I am filled with joy. I hope to continue to grow an inclusive and inviting community for years to come and help as many people as I can to follow their dreams.

Becoming a software engineer isn’t for everyone, but for those who love it as much as I do, it is an incredibly rewarding career with so many interesting directions to go in and opportunities to take. I’d never say I regret my time as a video editor; getting to create content seen on ESPN and NFL Network was always a dream of mine. We can have more than one dream in our lives, and I’m excited to continue pursuing every one of mine.

I’d encourage anyone who is curious to at least take a prep class somewhere or do some research on their own to consider if this could be a good fit for them. I believe anyone could do it with the right mix of patience, perseverance, and desire. It is the kind of career you’ll be most successful in if you are truly passionate about it. But if you are interested in making the switch and need some guidance, please don’t hesitate to reach out.

**Comment by jhomar32** [comment]
report abuse

**Reply by AbdulRahmanIlyas** [reply]
Wha

**Comment by Annaelizabet** [comment]
I also want to do that for my app, how will you help me with that?

**Comment by AlecoDigitalCreator** [comment]
WOW!! 👍🏽 very nice.
Congratulations 👏🎉

**Comment by nivanov** [comment]
S be

**Comment by seregy1** [comment]
Мне бы найти свою нишу..

**Comment by enosV** [comment]
Super awesome story! I can relate except that I gave up on programming as a career... I'm not sure I can compete with AI is my main issue lol. Also I really enjoy being a systems admin and dabble with code on a few automation projects so its not all gone. But I applause persistence and patience, I know the struggle. Look forward to seeing your github grow.

**Reply by uzbakoV1975** [reply]
Привет

**Reply by Thanu555** [reply]
Hi I am you look good you missing something ....

**Comment by Binary-Ninja-007** [comment]
Nice Story!

**Comment by bangymci** [comment]
> I loved my job as a video editor.
> 
> I met incredibly cool celebrities, worked on really interesting projects, and had the opportunity to work at incredible companies. I started my career at the NFL, went to NBCUniversal, and lived the startup life after, freelance editing video for years for TheMuse, helping prospective employees find out more about employers they were interested in working for.
> 
> Yet, I wasn’t fulfilled.
> 
> There were a few reasons. The first was knowing you were replaceable at any moment. Who wouldn’t want to work for the NFL? On top of that, it just wasn’t sustainable. I found myself needing two or three jobs at a time to pay all my bills.
> 
> I needed a change.
> 
> It took five years of self-learning mixed with night classes and a bootcamp to make the switch, but finally, I became a software engineer.
> 
> For me, the right bootcamp was Codesmith. For no bigger reason other than I loved their teaching style and the community they had built. Not only was I able to hone my technical skills at a higher level, but I made friendships that have changed my life. For anyone interested in becoming a software engineer, a bootcamp is absolutely not necessary. But if you, like me, find it hard to stay focused and figure out the right next step to learn, they can be an incredibly helpful solution.
> 
> I often get asked about making the switch after 20/25/30/etc. I started to learn to code when I was 25, and I formally made the career change around the age of 30.
> 
> I try to be honest. As much as I shouldn’t have been, I used to be embarrassed to ask people a decade younger than myself for help at work. The career change taught me a lot of humility.
> 
> With that being said, the best part of making the career change was feeling like I was finally doing something I loved and that it had a sustainable future to it. There is so much room for growth and now, I work a second job teaching because I want to, not because I need to pay my bills.
> 
> I’ll never forget my past. Being a video editor means I understand clients, products, visual design, and keeping brand consistency really well. I understand how to take nothing and turn it into something beautiful that maintains a brand’s voice. Because of that experience, I am able to think from a completely different perspective than some of my teammates, and that in itself is an amazing thing.
> 
> Now that I’ve been in this career for a few years, I’m focused on giving back to the community and helping others who want to make a career change. I teach classes on weekends. I have a discord server where people can ask questions - technical or otherwise. Every time a former student sends me a message about accepting their first role in the industry, I am filled with joy. I hope to continue to grow an inclusive and inviting community for years to come and help as many people as I can to follow their dreams.
> 
> Becoming a software engineer isn’t for everyone, but for those who love it as much as I do, it is an incredibly rewarding career with so many interesting directions to go in and opportunities to take. I’d never say I regret my time as a video editor; getting to create content seen on ESPN and NFL Network was always a dream of mine. We can have more than one dream in our lives, and I’m excited to continue pursuing every one of mine.
> 
> I’d encourage anyone who is curious to at least take a prep class somewhere or do some research on their own to consider if this could be a good fit for them. I believe anyone could do it with the right mix of patience, perseverance, and desire. It is the kind of career you’ll be most successful in if you are truly passionate about it. But if you are interested in making the switch and need some guidance, please don’t hesitate to reach out.

**Comment by bangymci** [comment]
vivo y22

**Comment by bangymci** [comment]
I loved my job as a video editor.

I met incredibly cool celebrities, worked on really interesting projects, and had the opportunity to work at incredible companies. I started my career at the NFL, went to NBCUniversal, and lived the startup life after, freelance editing video for years for TheMuse, helping prospective employees find out more about employers they were interested in working for.

Yet, I wasn’t fulfilled.

There were a few reasons. The first was knowing you were replaceable at any moment. Who wouldn’t want to work for the NFL? On top of that, it just wasn’t sustainable. I found myself needing two or three jobs at a time to pay all my bills.

I needed a change.

It took five years of self-learning mixed with night classes and a bootcamp to make the switch, but finally, I became a software engineer.

For me, the right bootcamp was Codesmith. For no bigger reason other than I loved their teaching style and the community they had built. Not only was I able to hone my technical skills at a higher level, but I made friendships that have changed my life. For anyone interested in becoming a software engineer, a bootcamp is absolutely not necessary. But if you, like me, find it hard to stay focused and figure out the right next step to learn, they can be an incredibly helpful solution.

I often get asked about making the switch after 20/25/30/etc. I started to learn to code when I was 25, and I formally made the career change around the age of 30.

I try to be honest. As much as I shouldn’t have been, I used to be embarrassed to ask people a decade younger than myself for help at work. The career change taught me a lot of humility.

With that being said, the best part of making the career change was feeling like I was finally doing something I loved and that it had a sustainable future to it. There is so much room for growth and now, I work a second job teaching because I want to, not because I need to pay my bills.

I’ll never forget my past. Being a video editor means I understand clients, products, visual design, and keeping brand consistency really well. I understand how to take nothing and turn it into something beautiful that maintains a brand’s voice. Because of that experience, I am able to think from a completely different perspective than some of my teammates, and that in itself is an amazing thing.

Now that I’ve been in this career for a few years, I’m focused on giving back to the community and helping others who want to make a career change. I teach classes on weekends. I have a discord server where people can ask questions - technical or otherwise. Every time a former student sends me a message about accepting their first role in the industry, I am filled with joy. I hope to continue to grow an inclusive and inviting community for years to come and help as many people as I can to follow their dreams.

Becoming a software engineer isn’t for everyone, but for those who love it as much as I do, it is an incredibly rewarding career with so many interesting directions to go in and opportunities to take. I’d never say I regret my time as a video editor; getting to create content seen on ESPN and NFL Network was always a dream of mine. We can have more than one dream in our lives, and I’m excited to continue pursuing every one of mine.

I’d encourage anyone who is curious to at least take a prep class somewhere or do some research on their own to consider if this could be a good fit for them. I believe anyone could do it with the right mix of patience, perseverance, and desire. It is the kind of career you’ll be most successful in if you are truly passionate about it. But if you are interested in making the switch and need some guidance, please don’t hesitate to reach out.

**Comment by Bunyuh** [comment]
🖖https://www.cve.org/Downloads

**Comment by Sujanayasa** [comment]
Luar biasa hebat teman"
